src/types/modifications/FuelPrice.jl

```diff
@@ -9,12 +9,19 @@ FuelPrice is a [`Modification`](@ref) allowing users to specify fuel prices for
 * [`modify_results!(mod::FuelPrice, config, data)`](@ref)
 
 To adjust price by hour or year, see [`AdjustHourly`](@ref) or [`AdjustYearly`](@ref).
+
+## Fields 
+* `file` - file path to the table of fuel steps. See [`summarize_table(::Val{:fuel_price})`]@(ref)
+* `fp_scalar` - a `Float64` for how much to scale the fuel price variables by.  This helps with numerical instability, given that some fuel steps can be very large and could bloat the RHS and bounds range of the model.
```

Comment:
I think the only think I would change here is to show the default value of 1e3, and say some language like "the default value of 1e3 means that the fuel quantity variables are in terms of thousands of MMBtu rather than MMBtu."